utils/build/docker/nodejs/parametric/servicer.js

```diff
@@ -1,294 +0,0 @@
-        }
-        const extracted = tracer.extract('http_headers', convertedHeaders)
-        if (extracted !== null) parent = extracted
-
-        const span = tracer.startSpan(request.name, {
-            type: request.type,
-            resource: request.resource,
-            childOf: parent,
-            tags: {
-                service: request.service
-            }
-        })
-
-        this.spans[span.context().toSpanId()] = span
-
-        return callback(null, {
-            span_id: span.context().toSpanId(),
-            trace_id: span.context().toTraceId()
-        })
-    }
-
-    InjectHeaders = (call, callback) => {
-        const { request } = call
-        const span = this.spans[request.span_id]
-        const http_headersDict = {}
-        const http_headers = []
-
-        tracer.inject(span, 'http_headers', http_headersDict)
-        for (const [key, value] of Object.entries(http_headersDict)) {
-            http_headers.push({key: key, value: value})
-        }
-
-        return callback(null, {
-            http_headers: { http_headers }
-        })
-    }
-
-    SpanSetMeta = (call, callback) => {
-        const span = this.spans[call.request.span_id]
-        span.setTag(call.request.key, call.request.value)
-        return callback(null, {})
-    }
-
-    // dd-trace-js has support for numeric values in tags
-    SpanSetMetric = (call, callback) => {
-        return this.SpanSetMeta(call, callback)
-    }
-
-    SpanSetError = (request, callback) => {
-        const span = this.spans[request.span_id]
-        span.addTags({
-            'error.msg': request.message,
-            'error.type': request.type,
-            'error.stack': request.stack
-        })
-        return callback(null, {})
-    }
-
-    FinishSpan = (call, callback) => {
-        const { id } = call.request
-        const span = this.spans[id]
-        span.finish()
-        delete this.spans[id]
-        return callback(null, {})
-
-    }
-
-    FlushSpans = (_, callback) => {
-        const { _tracer: { _exporter: { _writer } } } = tracer
-        _writer.flush(() => {
-            callback(null, {})
-        })
-
-    }
-
-    FlushTraceStats = (_, callback) => {
-        // TODO: implement once available in NodeJS Tracer
-        return callback(null, {})
-    }
-
-    HTTPClientRequest = ({ request }, callback) => {
-        const http = require('http')
-
-        const options = {
-            method: request.method,
-            headers: request.headers.http_headers.reduce((prev, next) => {
-                prev[next.key] = next.value
-                return prev
-            }, {})
-        }
-        const req = http.request(request.url, options, res => {
-            res.on('data', () => {})
-            res.on('end', () => callback(null, { statusCode: res.statusCode }))
-        })
-        req.on('error', e => callback(e))
-        req.write(request.body)
-        req.end()
-    }
```

Comment:
Was this one intended to be removed?

Reply:
For now, yes. It can be added back in later, but it only was useful for a single test and didn't seem entirely necessary for conversion.

Reply:
I'll see if I can easily add it back in. If not, I don't think it should block.